traffic_ops/traffic_ops_golang/origin/origins.go

```diff
@@ -487,3 +488,295 @@ func deleteQuery() string {
 WHERE id=:id`
 	return query
 }
+
+// Get is the handler for GET requests to Origins of APIv5.
+func Get(w http.ResponseWriter, r *http.Request) {
+
+	var useIMS bool
+
+	inf, sysErr, userErr, errCode := api.NewInfo(r, nil, nil)
+	if sysErr != nil || userErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	origins, userErr, sysErr, errCode, _ := getOrigins(w.Header(), inf.Params, inf.Tx, inf.User, useIMS)
+
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+
+	returnable := make([]tc.OriginV5, 0) // Initialize as an empty slice
+
+	for _, origin := range origins {
+		returnable = append(returnable, origin.ToOriginV5())
+	}
```

Comment:
this works fine, but fwiw you can avoid reallocations because you know the size ahead of time like:
```go
returnable := make([]tc.OriginV5, len(origins))
for i, origin := range origins {
    returnable[i]=origin.ToOriginV5()
}
```